src/vs/workbench/api/common/extHostVariableResolverService.ts

```diff
@@ -100,20 +107,24 @@ class ExtHostVariableResolverService extends AbstractVariableResolverService {
 				return undefined;
 			},
 			getSelectedText: (): string | undefined => {
-				if (editorService) {
-					const activeEditor = editorService.activeEditor();
-					if (activeEditor && !activeEditor.selection.isEmpty) {
-						return activeEditor.document.getText(activeEditor.selection);
-					}
+				const selection = getSelection(editorService);
+				if (selection && !selection.isEmpty) {
+					return editorService?.activeEditor()?.document.getText(selection);
+
 				}
 				return undefined;
 			},
 			getLineNumber: (): string | undefined => {
-				if (editorService) {
-					const activeEditor = editorService.activeEditor();
-					if (activeEditor) {
-						return String(activeEditor.selection.end.line + 1);
-					}
+				const selection = getSelection(editorService);
+				if (selection) {
+					return String(selection.end.line + 1);
+				}
+				return undefined;
+			},
+			getColumnNumber: (): string | undefined => {
+				const selection = getSelection(editorService);
+				if (selection) {
+					return String(selection.end.character + 1);
```

Comment:
Just getting the end character doesn't actually give the column if there are characters that span more than one column, like tabs.